src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,228 @@
+
+.. index:: ! truncate
+
+.. admonition:: Tip
+
+   Notice that existing events will not have a category, which violates our new ``@NotNull`` validation rule. The easiest way to handle this is to delete all rows in ``event``.
+   
+   To delete all events, truncate the ``event`` table in MySQL Workbench. (To truncate a SQL table is simply to delete all its rows.) To do this, select the table in the *Schemas* pane, right-click, and select *Truncate Table...*
+
+Start up the application and refresh the view in MySQL Workbench. 
+
+.. figure:: figures/event-table-after-update.png
+   :alt: The event table after starting up the application
+   :width: 600px
+
+   The ``event`` retains the ``type`` column, but has a new ``event_category_id`` column
+
+Notice that there is a new column, ``event_category_id``. This new column has type ``int`` and is a foreign key column to the ``event_category`` table. References to objects in Java are translated into foreign-key references by Hibernate. 
+
+.. admonition:: Note
+
+   You may find it odd that the ``type`` column is *still* present, even after restarting. Hibernate will not drop columns from a table even if that field is removed from the corresponding class. 
+
+   Hibernate will generally avoid deleting your data, since you may want to keep it around (even if just in the form of a backup). You can simply drop this column yourself.
+
+.. admonition:: Tip
+
+   If your table was not updating upon restarting, make sure you have ``spring.jpa.hibernate.ddl-auto`` set to ``update`` in ``application.properties``.
+
+If we create some new events, we can see precisely how ``event`` rows are related to ``event_category`` rows.
+
+.. figure:: figures/event-table-with-data.png
+   :alt: The event table with two rows
+
+   The ``event`` table with new data
+
+.. figure:: figures/event-category-table-with-data.png
+   :alt: The event_category table with two rows
+
+
+   The ``event_category`` table 
+
+Notice that our events have ``event_category_id`` values of ``12`` and ``13`` (these values may differ slightly in your application). These are foreign keys into the ``event_category`` table. For example, in the tables above, the ``event`` row with ``id`` equal to ``14`` is related to the ``event_category`` row with ``id`` equal to ``12``. This database relationship corresponds to the relationship between objects that was created by Spring Boot when we submitted the form.
+
+With this many-to-one relationship in place, we next look at how to establish a persistent one-to-many relationship.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   Name the annotation on ``AbstractEntity`` to allow it handle primary key id generation of subclasses but not map to a database 
```

Comment:
"...to allow it _to_ handle..."

I would also rephrase a bit for clarity, and phrase as a question: "What is the annotation that allows `@AbstractEntity` to handle logic related to IDs/primary keys of subclasses, yet not be mapped to a database table."

(Or something like that...the language feels a bit awkward almost any way you put it.)